actors/core/src/main/java/cloud/orbit/actors/cluster/NodeAddressImpl.java

```diff
@@ -68,4 +75,21 @@ public String toString()
     {
         return address.toString();
     }
+
+    @Override
+    public int compareTo(@NotNull final NodeAddress o)
```

Comment:
Technically not necessary at all. However, this does make inspecting either `ClusterView` or `RedisClusterTracker` more pleasant as it enforces an ordering on `SortedMap<NodeAddress, ?>`